src/main/java/blusunrize/lib/manual/Tree.java

```diff
@@ -143,11 +161,32 @@ public NT getNodeData()
 		}
 
 		public InnerNode<NT, LT> getOrCreateSubnode(NT data, int weight)
+		{
+			return getOrCreateSubnode(data, () -> weight);
+		}
+
+		public InnerNode<NT, LT> getOrCreateSubnode(NT data, DoubleSupplier weight)
+		{
+			return getSubnode(data)
+					.orElseGet(() -> addNewSubnode(data, weight));
```

Comment:
This line seems short and simple enough to not require a break. Any reason here, something I should improve about my style as well?

Reply:
I think this may have been more complex before, and I forgot to remove the line break after simplifying it. So no real reason.